.github/workflows/ci.yml

```diff
@@ -43,8 +43,28 @@ jobs:
       - name: Test package
         run: docker run --rm ${{ env.TEST_IMAGE }} "poe test"
 
-      # TODO setup coverage in #8
-#      - name: Run codacy coverage reporter
-#        uses: codacy/codacy-coverage-reporter-action@v1
-#        with:
-#          project-token: ${{ secrets.CODACY_PROJECT_TOKEN }}
+  docs-test:
+    name: Test documentation
```

Comment:
is it the documention of Tests or the test of documentation ? in the steps I see mkdocs build "only" does it include running tests ?

Reply:
it is a step that checks "tests" that documentation site will build without an issue.  Thus, the `build` command if it runs without errors, it means that the docs will be able to be published for the gh-pages branch